src/groovy/org/pih/warehouse/core/PriceTypeCode.groovy

```diff
@@ -0,0 +1,13 @@
+package org.pih.warehouse.core
+
+enum PriceTypeCode {
+    AVERAGE_COST,
+    COMPETITIVE_PRICE,
```

Comment:
Let's cut these to just the ones we need for now. Probably just DEFAULT_PRICE, AVERAGE_COST. But ask Kelsey for her thoughts.

Reply:
fixed